docs/wiki/installation/install-osx.md

```diff
@@ -43,6 +43,16 @@ To update osquery:
 $ brew update
 $ brew upgrade osquery
 ```
+Post installation steps to run osqueryd if you have never installed and run osqueryd on this Mac.
```

Comment:
This sentence is a bit odd, is it meant to be a heading? Maybe: `#### Post installation steps` then the first sentence in the section can read: this only applies to the first time you're setting up `osqueryd` on this Mac.